app/index.jsx

```diff
@@ -22,15 +22,17 @@ $("#collapseButton").on("click", function() {
 
   if (dataP.hasClass("panel-collapsed") && templateV.hasClass("panel-collapsed")) {
 
-    dataP.animate({ "width": "20%"}, animationDuration, animationEasing, setCollapse(dataP, false));
-    templateV.animate({ width: "20%"}, animationDuration, animationEasing, setCollapse(templateV, false));
+    dataP.animate({ "width": "20%", opacity: "1.0"}, animationDuration, animationEasing, setCollapse(dataP, false));
+    templateV.animate({ width: "20%", opacity: "1.0"}, animationDuration, animationEasing, setCollapse(templateV, false));
     visualizerW.animate({width: "60%"});
+    $("#collapseButton").text("Collapse");
 
     } else {
 
-    dataP.animate({ width: "0%"}, animationDuration, animationEasing, setCollapse(dataP, true));
-    templateV.animate({ width: "0%"}, animationDuration, animationEasing, setCollapse(templateV, true));
+    dataP.animate({ width: "0%", opacity: "0.0"}, animationDuration, animationEasing, setCollapse(dataP, true));
+    templateV.animate({ width: "0%", opacity: "0.0"}, animationDuration, animationEasing, setCollapse(templateV, true));
     visualizerW.animate({width: "100%"});
+    $("#collapseButton").text("Expand");
```

Comment:
I don't think we should use the hyphen.  Since css uses a lot of hyphenated variable names by default, it's usually bad practice to use hyphens in CSS classes and IDs.  I'd vote camel case or underscore.